docs/howto/operate/new-hub/index.md

```diff
@@ -61,21 +62,26 @@ To deploy a new hub, follow these steps:
 8. Log in to the hub and ensure that the hub works as expected from a user's perspective.
 9. Send a link to the hub's Community Representative(s) so they can confirm that it works from their perspective as well.
 
+:::{note}
+If you need to deploy a new hub in a AWS cluster, there are some minor additional steps described on this [section](new-hub:aws).
+:::
+
 ## Automated vs. manual deploys
 
 Some of our infrastructure automatically deploys and updates hubs via GitHub Workflows, while others require manual deploys.
 This is changing over time as we automate more things, and is dependent on the cloud provider.
 
-The best place to look to learn about the latest state of our *automatic* hub deployment is to look at [the `deploy-hubs.yaml` GitHub workflow](https://github.com/2i2c-org/pilot-hubs/blob/master/.github/workflows/deploy-hubs.yaml).
-That workflow defines the automatic hub deployment for many of our major clusters.
+[](operate:ci-cd)
 
-See the sections below for information about deploying hubs on each cloud provider, including whether to manually or automatically deploy.
+You will find more information about our automation in [](operate:ci-cd)
 
 ## Deploying hubs manually
 
 Some of our infrastructure still requires manual deploys.
-The following sections cover how to deploy in these situations.
+There are also situations where you may want to deploy infrastructure manually.
```

Comment:
Just a note that I think once we get the AWS automation stuff set up, we should **heavily** discourage deploying anything manually. I feel like every manual step is a large increase in the likelihood of confusion, bottlenecks of information, inconsistent state on the hub vs. the codebase, etc.